Content.Server/Content.Server.csproj

```diff
@@ -158,6 +159,10 @@
       <Project>{59250baf-0000-0000-0000-000000000000}</Project>
       <Name>Lidgren.Network</Name>
     </ProjectReference>
+    <ProjectReference Include="..\engine\SS14.Client\SS14.Client.csproj">
```

Comment:
Uhm? You're making the content server reference the client here.

Reply:
Whoops, this was definitely accidental... I didn't notice.